TalkAloud/TalkAloud/Managers/CoreDataManager.swift

```diff
@@ -99,6 +125,33 @@ class CoreDataManager {
         }
     }
     
+    func updateAudioRecordingFileName(with selectedRecording: AudioRecording, newFileName: String) {
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+
+        let managedContext = appDelegate.persistentContainer.viewContext
+        let changeRequest = NSFetchRequest<NSFetchRequestResult>(entityName: "AudioRecordingObject")
+
+        do {
+            let objects = try managedContext.fetch(changeRequest)
+            
+            for object in objects {
+                let recordingInObjects = AudioRecording(object: object as! NSManagedObject)
```

Comment:
Why not the same variable name like in the others?

Reply:
I thought I fixed that 😅